src/app/component/eco-news/components/create-news/create-news.component.ts

```diff
@@ -62,6 +62,12 @@ export class CreateNewsComponent implements OnInit {
     }
   }
 
+  private autoResize(event) {
```

Comment:
Please, change the acces modifier to 'public' because of AoT

Reply:
fixed